### ISSUE TITLE
Including info from your presentation

### ISSUE THREAD

**Issue by alecbaines** [calvincfso/OFF-ALPACA-Caecilian#1]
This repo is very thorough but I would like to have seen a couple sentences in your conclusions section about what kinds of scenarios ALPACA would be more suited for. I really liked how you discussed in your presentation various reasons why ALPACA worked better for other studies like the mice paper and I think that people outside of this class that use this repo would benefit from it too.

**Comment by MorphoFun** [comment]
> This repo is very thorough but I would like to have seen a couple sentences in your conclusions section about what kinds of scenarios ALPACA would be more suited for. I really liked how you discussed in your presentation various reasons why ALPACA worked better for other studies like the mice paper and I think that people outside of this class that use this repo would benefit from it too.

I agree! This is a really helpful guide for anyone looking to apply ALPACA to specimens other than mice. It's not uncommon for computer programs to work great on datasets that look similar to those they were trained on but then struggle more if the samples are way different from the training dataset.
